docs/FAKE_DISCORD_SERVER.md

```diff
@@ -0,0 +1,57 @@
+# Beware of Fake Discord Impersonating SkyHanni
+
+
+## What is happening?
+
+We've discovered a fake Discord server pretending to be SkyHanni.
+This fake server looks very similar to ours, with more than 15,000 users, with similar channel names and our profile picture.
+They even have a user named "hannibal2."
+
+It's crucial to know that this fake server has nothing to do with the real SkyHanni server.
+
+The people behind this fake server want to steal your minecraft account!
+They will ask you to verify your identity by clicking on a Microsoft link to access chat channels.
+
+To keep yourself safe, please don't interact with this fake server, and leave it immediately.
+
+## Why dont we do anything about them?
+
+We have successfully reported and taken down the server in the past.
+But the server got always recreated shortly after.
+Sadly the Discord support team has stopped responding after a while.
+
+## I have got hacked / my account got stolen. What to do?
+
+### I cant log in into my account anymore
+
+You can try to get your account back by contacting microsoft support.
+You need to send them a ticket via your email, telling them what happened.
+Also provide them with exact information how and when you have bought your minecraft account.
+Good luck!
+
+### Someone else logged in into my account, but i can still access the account myself
+
+Disable the multiplayer mode on xbox account settings to prevent others from logging in your account immediately.
+Try to log in on hypixel to kick the other user off from hypixel.
+Change your password on minecraft.net.
+Wait 24 hours and enable multiplayer mode again.
+
+## How to detect the fake server?
+
+> The fake server asks you to verify via entering data.
+
+The real server will never ask you to enter your account name and password.
+
+> The fake server has many channels without any sent message sent or without access to see the messages in the channel
+
+The real server has a lot of channels you can view and everyone of them has a lot of messages that (for some channels) even go back over a year.
+
+> The fake server does not link to anyhting outside itself.
+
+The real server has channels that link to github, to updates, to other ressources etc.
+
+> The fake server does not allow you to write in the chat.
```

Comment:
I think this is misleading as I do believe you can type in the channels (though some may be locked behind "verification") but the chat history is a good point